File: co.edu.uniandes.fixml.toCSharp/test/test3.xml/ApplicationMessage.cs

using System.Collections.Generic;
// Class for tag ApplicationMessage
class ApplicationMessage {


	List<Order> _Orders = new List<Order>();
	
	// default constructor
	
	public ApplicationMessage ( ) {   
		;
		
		Order ApplicationMessage_Order= new Order();
		ClOrdID ApplicationMessage_Order_ClOrdID= new ClOrdID();
		ApplicationMessage_Order.AddClOrdID( ApplicationMessage_Order_ClOrdID );
		
		HandInst ApplicationMessage_Order_HandInst= new HandInst("2");
		ApplicationMessage_Order.AddHandInst( ApplicationMessage_Order_HandInst );
		
		MinQty ApplicationMessage_Order_MinQty= new MinQty();
		ApplicationMessage_Order.AddMinQty( ApplicationMessage_Order_MinQty );
		
		Instrument ApplicationMessage_Order_Instrument= new Instrument();
		Symbol ApplicationMessage_Order_Instrument_Symbol= new Symbol();
		ApplicationMessage_Order_Instrument.AddSymbol( ApplicationMessage_Order_Instrument_Symbol );
		
		IDSource ApplicationMessage_Order_Instrument_IDSource= new IDSource();
		ApplicationMessage_Order_Instrument.AddIDSource( ApplicationMessage_Order_Instrument_IDSource );
		
		SecurityID ApplicationMessage_Order_Instrument_SecurityID= new SecurityID();
		ApplicationMessage_Order_Instrument.AddSecurityID( ApplicationMessage_Order_Instrument_SecurityID );
		
		ApplicationMessage_Order.AddInstrument( ApplicationMessage_Order_Instrument );
		
		Side ApplicationMessage_Order_Side= new Side("1");
		ApplicationMessage_Order.AddSide( ApplicationMessage_Order_Side );
		
		TransactTime ApplicationMessage_Order_TransactTime= new TransactTime();
		ApplicationMessage_Order.AddTransactTime( ApplicationMessage_Order_TransactTime );
		
		OrderQuantity ApplicationMessage_Order_OrderQuantity= new OrderQuantity();
		OrderQty ApplicationMessage_Order_OrderQuantity_OrderQty= new OrderQty();
		ApplicationMessage_Order_OrderQuantity.AddOrderQty( ApplicationMessage_Order_OrderQuantity_OrderQty );
		
		ApplicationMessage_Order.AddOrderQuantity( ApplicationMessage_Order_OrderQuantity );
		
		OrderType ApplicationMessage_Order_OrderType= new OrderType();
		LimitOrder ApplicationMessage_Order_OrderType_LimitOrder= new LimitOrder("2");
		Price ApplicationMessage_Order_OrderType_LimitOrder_Price= new Price();
		ApplicationMessage_Order_OrderType_LimitOrder.AddPrice( ApplicationMessage_Order_OrderType_LimitOrder_Price );
		
		ApplicationMessage_Order_OrderType.AddLimitOrder( ApplicationMessage_Order_OrderType_LimitOrder );
		
		ApplicationMessage_Order.AddOrderType( ApplicationMessage_Order_OrderType );
		
		Currency ApplicationMessage_Order_Currency= new Currency("USD");
		ApplicationMessage_Order.AddCurrency( ApplicationMessage_Order_Currency );
		
		Rule80A ApplicationMessage_Order_Rule80A= new Rule80A("A");
		ApplicationMessage_Order.AddRule80A( ApplicationMessage_Order_Rule80A );
		
		this.AddOrder( ApplicationMessage_Order );
		
		
	}
	
	public ApplicationMessage (  ) {
	
	}
	
	public void AddOrder( Order child ) {
		this._Orders.Add( child );
	}
	
	
} 

File: co.edu.uniandes.fixml/test/co/edu/uniandes/fixml/src-gen/test3/Java/ApplicationMessage.java
package object;

import java.util.List;
import java.util.ArrayList;

// Class for tag ApplicationMessage
public class ApplicationMessage {


  List<Order> Orders = new ArrayList<Order>();



  public ApplicationMessage ( ) {   
  
;
  
Order ApplicationMessage_Order_0= new Order();
ClOrdID ApplicationMessage_Order_0_ClOrdID_0= new ClOrdID();
ApplicationMessage_Order_0.addClOrdID( ApplicationMessage_Order_0_ClOrdID_0 );
  		
HandInst ApplicationMessage_Order_0_HandInst_1= new HandInst();
ApplicationMessage_Order_0.addHandInst( ApplicationMessage_Order_0_HandInst_1 );
  		
MinQty ApplicationMessage_Order_0_MinQty_2= new MinQty();
ApplicationMessage_Order_0.addMinQty( ApplicationMessage_Order_0_MinQty_2 );
  		
Instrument ApplicationMessage_Order_0_Instrument_3= new Instrument();
Symbol ApplicationMessage_Order_0_Instrument_3_Symbol_0= new Symbol();
ApplicationMessage_Order_0_Instrument_3.addSymbol( ApplicationMessage_Order_0_Instrument_3_Symbol_0 );
  		
IDSource ApplicationMessage_Order_0_Instrument_3_IDSource_1= new IDSource();
ApplicationMessage_Order_0_Instrument_3.addIDSource( ApplicationMessage_Order_0_Instrument_3_IDSource_1 );
  		
SecurityID ApplicationMessage_Order_0_Instrument_3_SecurityID_2= new SecurityID();
ApplicationMessage_Order_0_Instrument_3.addSecurityID( ApplicationMessage_Order_0_Instrument_3_SecurityID_2 );
  		
ApplicationMessage_Order_0.addInstrument( ApplicationMessage_Order_0_Instrument_3 );
  		
Side ApplicationMessage_Order_0_Side_4= new Side();
ApplicationMessage_Order_0.addSide( ApplicationMessage_Order_0_Side_4 );
  		
TransactTime ApplicationMessage_Order_0_TransactTime_5= new TransactTime();
ApplicationMessage_Order_0.addTransactTime( ApplicationMessage_Order_0_TransactTime_5 );
  		
OrderQuantity ApplicationMessage_Order_0_OrderQuantity_6= new OrderQuantity();
OrderQty ApplicationMessage_Order_0_OrderQuantity_6_OrderQty_0= new OrderQty();
ApplicationMessage_Order_0_OrderQuantity_6.addOrderQty( ApplicationMessage_Order_0_OrderQuantity_6_OrderQty_0 );
  		
ApplicationMessage_Order_0.addOrderQuantity( ApplicationMessage_Order_0_OrderQuantity_6 );
  		
OrderType ApplicationMessage_Order_0_OrderType_7= new OrderType();
LimitOrder ApplicationMessage_Order_0_OrderType_7_LimitOrder_0= new LimitOrder();
Price ApplicationMessage_Order_0_OrderType_7_LimitOrder_0_Price_0= new Price();
ApplicationMessage_Order_0_OrderType_7_LimitOrder_0.addPrice( ApplicationMessage_Order_0_OrderType_7_LimitOrder_0_Price_0 );
  		
ApplicationMessage_Order_0_OrderType_7.addLimitOrder( ApplicationMessage_Order_0_OrderType_7_LimitOrder_0 );
  		
ApplicationMessage_Order_0.addOrderType( ApplicationMessage_Order_0_OrderType_7 );
  		
Currency ApplicationMessage_Order_0_Currency_8= new Currency();
ApplicationMessage_Order_0.addCurrency( ApplicationMessage_Order_0_Currency_8 );
  		
Rule80A ApplicationMessage_Order_0_Rule80A_9= new Rule80A("A");
ApplicationMessage_Order_0.addRule80A( ApplicationMessage_Order_0_Rule80A_9 );
  		
this.addOrder( ApplicationMessage_Order_0 );
  		
  }
      
  public void addOrder( Order child ) {
    this.Orders.add( child );
  }

} 